src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -324,17 +324,18 @@ export function ChangeDataView({
           }}
           onCreateDefaultAdHocDataView={onCreateDefaultAdHocDataView}
         />
-      </>
+      </React.Fragment>
     );
 
     if (textBasedLanguages?.length) {
       panelItems.push(
-        <EuiHorizontalRule margin="none" />,
+        <EuiHorizontalRule margin="none" key="textbasedLanguages-divider" />,
```

Comment:
you probably need one last key in the `TextBasedLanguagesList` item pushed into the same array and that covers the SQL data views

Reply:
Added https://github.com/elastic/kibana/pull/148296/commits/c688932a39d4fffe7321164a4094122aab5f7514